microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -73,27 +63,27 @@ You can now run a {microshift-short} cluster with low-latency response rates. Us
 ==== Support for workload partitioning
 With this release, workload partitioning is supported in {microshift-short}. See xref:../microshift_configuring/microshift_low_latency/microshift-workload-partitioning.adoc#microshift-workload-partitioning[Workload partitioning] for more information.
 
-//[id="microshift-4-17-support_{context}"]
-//=== Support
-
 [id="microshift-4-17-doc-enhancements_{context}"]
 === Documentation enhancements
-//Doc enhancements include additions for RFEs and other continuous improvement items that are substantial and are not tied to new features or bug fixes already listed here. These can also go under the relevant section as applicable.
 
 [id="microshift-4-17-data-cleanup_{context}"]
 ==== Data cleanup now documented
 With this release, the `microshift-cleanup-data` script is documented. See xref:../microshift_troubleshooting/microshift-cleanup-data.adoc#microshift-cleanup-data[Data cleanup] for more information.
 
+[id="microshift-4-17-install-topics-reorg_{context}"]
+==== Installation topics are reorganized
+With this release, system requirements, installation types, and information about installing add-ons are reorganized into discrete topics so that they are easier to use.
+
 [id="microshift-4-17-tech-preview_{context}"]
-== Technology preview features
+== Technology Preview feature
 
-Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red{nbsp}Hat Customer Portal for these features:
+One feature in this release is currently in Technology Preview. This experimental feature is not intended for production use. Note the following scope of support on the Red{nbsp}Hat Customer Portal for this feature:
 
 link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
 
 [id="microshift-4-17-rhel-image-mode_{context}"]
 === {op-system-base-full} image mode Technology Preview feature
-* You can now install {microshift-short} using a `bootc` container image. Image mode for {op-system} is a technology preview deployment method that uses a container-native approach to build, deploy and manage the operating system as a bootc container image.
+* You can now install {microshift-short} using a `bootc` container image. Image mode for {op-system} is a Technology Preview deployment method that uses a container-native approach to build, deploy and manage the operating system as a bootc container image.
```

Comment:
Possibly `{op-system}` is missing from the MicroShift attributes file? it's showing up unrendered in the preview:
![image](https://github.com/user-attachments/assets/b7880b41-ace6-4d94-a65a-2291610e20d8)